node/silkworm/downloader/internals/db_utils.hpp

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2021-2022 The Silkworm Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#ifndef SILKWORM_DB_UTILS_H
+#define SILKWORM_DB_UTILS_H
```

Comment:
I believe in previous PR we have decided to uniform include guards with `#pragma once` only

Reply:
done